helper.py

```diff
@@ -0,0 +1,234 @@
+plus = [
+  "    ___    ",
+  "   |   |   ",
+  " __|   |__ ",
+  "|         |",
+  "|__     __|",
+  "   |   |   ",
+  "   |___|   ",
+]
+space = [
+  "  ",
+  "  ",
+  "  ",
+  "  ",
+  "  ",
+  "  ",
+  "  ",
+]
+number_one = [
+  "  ____   ",
+  " /    |  ",
+  "/_/|  |  ",
+  "   |  |  ",
+  "   |  |  ",
+  "  _|  |_ ",
+  " |______|",
+]
+for one in number_one:
+  print(one)
+
+number_two = [
+"   _____    ",
+" /  __  |  ",
+"/__/ / /   ",
+"    / /    ",
+"   / /     ",
+"  / /____   ",
+" |_______|  ",
+]
+for two in number_two:
+  print(two)
+
+number_three = [
+"  ________    ",
+" |_____  |   ",
+"    ___| |   ",
+"   |___  |   ",          
+"       | |   ",    
+"  _____| |   ",
+" |_______|   ", 
+]
+for three in number_three:
+  print(three)
+import ipdb
+number_four = [
+" __    __  ",
+"|  |  |  | ",
+"|  |  |  | ",
+"|  |__|  | ",
+"|_____|  | ",
+"      |  | ",
+"      |__| ",
+]
+for four in number_four:
+  print(four)
+
+number_five = [
+  " ______   ",
+  "|  ____|  ",
+  "|  |__    ",
+  "|____ \\  ",
+  " _   \\ | ",
+  "| |__| |  ",
+  " \____/   ",
+]
+for five in number_five:
+  print(five)
+
+number_six = [
+"  _______  ",
+  "|  ____| ",
+  "|  |__    ",
+  "| __  \\  ",
+  "| | \\  | ",
+  "| |__| |  ",
+  " \____/   ",
+]
+for six in number_six:
+  print(six)
+
+number_seven = [
+"  _______    ",
+" /____  /    ",
+"     / /     ",
+"    / /      ",
+"   / /       ",
+"  / /        ",
+" /_/         ",
+]
+
+for seven in number_seven:
+  print(seven)
+
+number_eight = [
+  "  _____  ",
+  " /  _  \\ ",
+  "/  |_|  \\",
+  "\\       /",
+  "/   _   \\",
+  "\\  |_|  /",
+  " \\_____/ ",
+]
+
+for eight in number_eight:
+  print(eight)
+
+number_nine = [
+  "  _______ ",
+  " /   _   |",
+  "|   |_|  |",
+  " \\___    |",
+  "     |   |",
+  "     |   |",
+  "     |___|",
+]
+
+for nine in number_nine:
+  print(nine)
+
+number_zero = [
+
+"   ____     ",
+"  / __ \\   ",
+" | /  \\ |  ",
+" | |  | |  ",
+" | |  | |  ",
+" | \\__/ |  ",
+"  \\____/   ",
+]
+for zero in number_zero:
+  print(zero)
+
+my_dict ={ "1" : [
+  "  ____   ",
+  " /    |  ",
+  "/_/|  |  ",
+  "   |  |  ",
+  "   |  |  ",
+  "  _|  |_ ",
+  " |______|",
+], "2" : [
+"   _____    ",
+" /  __  |  ",
+"/__/ / /   ",
+"    / /    ",
+"   / /     ",
+"  / /____   ",
```

Comment:
You are repeating code here:

The list-numbers are already defined above.

For example

```python
number_one = [
  "  ____   ",
  " /    |  ",
  "/_/|  |  ",
  "   |  |  ",
  "   |  |  ",
  "  _|  |_ ",
  " |______|",
]

...
# Rest of the numbers
...

number_nine = [
  "  _______ ",
  " /   _   |",
  "|   |_|  |",
  " \\___    |",
  "     |   |",
  "     |   |",
  "     |___|",
]


my_dict ={ 
"1" : number_one,
...
# follow the parent here and write the rest of the numbers.
....
"9":number_nine  
}


```